### ISSUE TITLE
FluidDissipation in Modelica.Fluid.Dissipation

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#1212]
**Reported by hansolsson on 16 Jul 2013 07:53 UTC**
In adcf4a26f3226877a9c153f79bf88384da950c93:
function Modelica.Fluid.Dissipation.PressureLoss.Bend.dp_edgedOverall_MFLOW

has
```
SI.Velocity v_lam= (2*(d_hyd/IN_var.eta)^exp/(A*C1*zeta_LOC*B*(IN_var.rho)^(1 - exp)))^(1/pow)*
      FluidDissipation.Utilities.Functions.General.SmoothPower(
      abs(dp),
      dp_min,
      1/pow) "Mean velocity under laminar conditions";
```
Note: the use of FluidDissipation.

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/1212

**Comment by modelica-trac-importer** [comment]
**Comment by dietmarw on 16 Jul 2013 08:28 UTC**
It also looks like lots of test scripts in `ModelicaTest` were never updated to `Fluid.Dissipation` and hence never get called.

**Comment by modelica-trac-importer** [comment]
**Comment by wischhusen on 16 Jul 2013 08:35 UTC**
The scripts have been used to generate figures for documentation. Perhaps they could be used again in future?

**Comment by modelica-trac-importer** [comment]
**Comment by wischhusen on 16 Jul 2013 08:49 UTC**
Updated the path of Utility function Stepsmoother with 6c643e7e702ee4db7d00e3bb7078eb4e65ac01af.

**Comment by modelica-trac-importer** [comment]
**Comment by otter on 17 Jul 2013 08:52 UTC**
Replying to [comment:1 dietmarw]:


> It also looks like lots of test scripts in `ModelicaTest` were never updated to `Fluid.Dissipation` and hence never get called.

Fixed in 03e15b26b64257a406d258abcdcb90924d3f6b96 and c1c2c71ecef705ba338ac9db2cff4f303b5bea2b:

- Removed scripts in directory that refer to models not available in ModelicaTest (these scripts contained the wrong FluidDissipation name).
- Removed cd(".\simulation") statements in the remaining scripts and instead
  introduced calls to inquire current working directory, change to a temporary
  directory from the operating system (%temp%) and at the end switch back
  to the initial working directory.
- Run all the scripts and they seem to work properly.
